web/app.js

```diff
@@ -812,6 +812,8 @@ const PDFViewerApplication = {
    *                      destruction is completed.
    */
   async close() {
+    this._unblockDocumentLoadEvent();
```

Comment:
While this *really shouldn't* matter for the Firefox built-in PDF Viewer use-case, since only the GENERIC viewer has openFile-support, in nonetheless can't hurt to ensure that closing a partially loaded document will actually fire the "load" event.